include/transport/transport.h

```diff
@@ -52,7 +52,7 @@ enum class TransportError : uint8_t
 /**
  * Transport Protocol to use
  */
-enum class TransportProtocol
+enum class TransportProtocol: uint8_t
```

Comment:
Qualify the underlying type to make as small as possible - the enumerations above this already did this 👍🏻